src/index.css

```diff
@@ -28,3 +28,44 @@
     filter: invert(1);
   }
 }
+
+.data-dictionary p {
+  margin: 0.5em 0;
+  padding: 0
+}
+
+.data-dictionary main h3 {
+  font-weight: bold;
+  font-size: 1.1em;
+  margin-top: 0.5em;
+}
+
+.data-dictionary cite, .data-dictionary .literal {
+  font-family: monospace;
+  font-style: normal;
+}
+
+.data-dictionary dl {
+  display: grid;
+  grid-gap: 0px 8px;
+  grid-template-columns: max-content;
+}
+
+.data-dictionary dt {
+  text-align: right;
+}
+
+.data-dictionary dd {
+  margin: 0;
+  grid-column-start: 2;
+}
```

Comment:
I lifted this from the wizard because i like the compact display it creates, but if having a grid in the search results makes things complicated we can rethink it